packages/kubernetes/manifest.yml

```
@@ -35,6 +35,14 @@ policy_templates:
         title: Collect Kubernetes metrics from Kubelet API
         description: Collecting Node, Pod, Container, Volume and System metrics from Kubelet
         input_group: metrics
+        vars:
+          - name: condition
+            title: Condition
+            description: Condition to filter when to collect this input. See [Dynamic Input Configuration](https://www.elastic.co/guide/en/fleet/current/dynamic-input-configuration.html) for details.
+            type: text
+            multi: false
+            required: false
+            show_user: false
```

```suggestion
            show_user: true
```

The space is occupied and I would like to leave this open for users to see what inside `Settings`